### PR TITLE
Removes singularity in favour of twin TEG's

### PR DESCRIPTION
**Pull request by optimumtact**
Removes singularity and replaces with a less damaging engine type that will prevent so many rounds ending as early.

### CODE REVIEW

**Comment by optimumtact** [comment]
screenshots coming

**Comment by Lumipharon** [comment]
supermatter would be better, but anything is better then LAG MASTER 5000

**Comment by kyrahabattoir** [comment]
Why not both?

**Comment by Profakos** [comment]
The code only has changes to the centcomm zlevel. Is this the correct branch?

**Comment by Dorsisdwarf** [comment]
This is definitely the wrong branch, seeing as the last commit was two months ago for a completely different thing to the title and post.

Alternatively, oranges is being oranges

**Comment by GunHog** [comment]
100% Against :-1: Do not remove Our Lord!

**Comment by Fox-McCloud** [comment]
So the singularity beacon is now useless and engineers have lost their single specialized tool they had.

Not to mention that the TeG will literally never fail or have a single bad side effect.

**Comment by Stickymayhem** [comment]
Oranges is just being silly. It's a few neat changes to centcom.

File: _maps/map_files/generic/z2.dmm [hunk 1]
```diff
@@ -354,12 +354,17 @@
 "gP" = (/obj/structure/optable,/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/space)
 "gQ" = (/obj/structure/table,/obj/item/weapon/circular_saw,/obj/item/weapon/cautery,/obj/item/weapon/surgicaldrill,/obj/item/robot_parts/l_arm,/obj/item/robot_parts/r_arm,/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/space)
 "gR" = (/obj/structure/table,/obj/item/weapon/scalpel,/obj/item/weapon/retractor,/obj/item/weapon/hemostat,/obj/item/weapon/surgical_drapes,/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/space)
+<<<<<<< HEAD
```
Comment (HoaxIT) [inline _maps/map_files/generic/z2.dmm]: :+1: We totally need this!

**Comment by WJohn** [comment]
Can the TEG at least explode or something? It needs to have a catastrophic failure, but one that's not as damaging as the singulo (which in theory can devour every piece of the station.

**Comment by GunHog** [comment]
WJ, it is a troll, read the files!

**Comment by WJohn** [comment]
Reading files is for nerds.

**Comment by SconesC** [comment]
it was a good joke though

**Comment by optimumtact** [comment]
Dorsisdwarf is a fun pirate

**Pull request closed without merging**